LauncherInjector/main.cpp

```diff
@@ -345,10 +367,12 @@ int main(int argc, char* argv[])
 	auto LauncherMain = GetLauncherMain();
 	if (!LauncherMain)
 		MessageBoxA(
-			GetForegroundWindow(), "Failed loading launcher.dll.\nThe game cannot continue and has to exit.", "Northstar Launcher Error",
-			0);
+			GetForegroundWindow(), "Failed loading launcher.dll.\nThe game cannot continue and has to exit.",
+			"Northstar Launcher Error", 0);
 	// auto result = ((__int64(__fastcall*)())LauncherMain)();
 	// auto result = ((signed __int64(__fastcall*)(__int64))LauncherMain)(0i64);
-	return ((int(/*__fastcall*/*)(HINSTANCE, HINSTANCE, LPSTR, int))LauncherMain)(
-		NULL, NULL, NULL, 0); // the parameters aren't really used anyways
+	printf("Starting normally");
```

Comment:
This is missing the newline and doesn't match the format of the other log messages.